jekyll/_cci2/server/v4.4/air-gapped-installation/phase-1-prerequisites.adoc

```diff
@@ -48,17 +49,17 @@ CircleCI server is deployed into a Kubernetes cluster using a Helm chart. You wi
 
 [#login-to-acr]
 === a. Log in to CircleCI server container registry
-Access to the images you need for your installation requires you to use a username and token, which will be provided by CircleCI. Please contact mailto:support@circleci.com[] for more information.
+Access to the images you need for your installation requires you to use a username and token, which will be provided by CircleCI. Contact mailto:support@circleci.com[CircleCI support] for more information.
```

Comment:
Can we change this from `mailto:` to a link to the support portal? 
https://support.circleci.com/hc/en-us

support@ no longer creates a ticket and will just autoreply to the email.

Reply:
Ah yes, I thought I had remove all these actually. Must have not searched in the server docs when I did that so will look for all instances. Thanks!